.ci/after_success.sh

```diff
@@ -4,8 +4,10 @@ set -ev
 
 bash <(curl -s https://codecov.io/bash)
 
+AVALANCHE_IMAGE=$(docker image ls --format="{{.Repository}}:{{.Tag}}" | grep $DOCKERHUB_REPO | head -n 1)
```

Comment:
Why do we do this this way? Why can there be more results for `docker image ls` which match so that we need to do a `head -n 1` and how do we know that this will always be the correct one?

Reply:
This should be built when `scripts/build_image.sh` is called in `.ci/run_e2e_tests.sh`. Previously, this relied on the image built by `.ci/run_docker.sh` script which built using the tag $TRAVIS_TAG, so it was identical. However, there was an issue due to the fact that the travis commit and the commit used by the build_image.sh script had a one character difference.

As long as there's only one avalanchego image built in CI, there will only be one match here. As for the terminology, I just borrowed the `head -n 1` syntax from the `.ci/run_e2e_tests.sh` script.

As long as we only build one docker image for avalanchego within CI, this will be unique, but to make it more explicit, I can try to change the build_image.sh script to take the same length substring from the commit when tagging the docker image.